tests/collectors/euctr/test_parser.py

```diff
@@ -8,18 +8,34 @@
 
 
 class TestEuctrParser(object):
-    def test_trial_results_returns_absolute_results_url(self, get_url):
+    def test_trial_results_url_returns_absolute_results_url(self, get_url):
         url = 'https://www.clinicaltrialsregister.eu/ctr-search/trial/2011-005852-33/3rd'
         response = get_url(url)
 
         record = parse_record(response)
 
-        assert record['trial_results'] == 'https://www.clinicaltrialsregister.eu/ctr-search/trial/2011-005852-33/results'
+        assert record['trial_results_url'] == 'https://www.clinicaltrialsregister.eu/ctr-search/trial/2011-005852-33/results'
 
-    def test_trial_results_is_none_if_therere_no_results(self, get_url):
+    def test_trial_results_url_is_none_if_therere_no_results(self, get_url):
         url = 'https://www.clinicaltrialsregister.eu/ctr-search/trial/2009-016529-32/EE'
         response = get_url(url)
 
         record = parse_record(response)
 
-        assert record.get('trial_results') is None
+        assert record.get('trial_results_url') is None
+
+    def test_trial_results_url_is_none_if_results_not_hyperlink(self, get_url):
+        url = 'https://www.clinicaltrialsregister.eu/ctr-search/trial/2005-002909-23/PT'
+        response = get_url(url)
+
+        record = parse_record(response)
+
+        assert record.get('trial_results_url') is None
+
+    def test_trial_results_returns_results_text(self, get_url):
+        url = 'https://www.clinicaltrialsregister.eu/ctr-search/trial/2005-002909-23/PT'
+        response = get_url(url)
+
+        record = parse_record(response)
+
+        assert record.get('trial_results') == 'Removed from public view';
```

Comment:
(This comment is for all the tests here)

Although a test should generally have only one assert (as you've done here), `trial_results` and `trial_results_url` are so intertwined that I think it would be good to add assertions for both in these tests.